docs/source/intro.rst

```diff
@@ -0,0 +1,44 @@
+===========
+ome-zarr-py
+===========
+
+Experimental support for multi-resolution images stored in Zarr filesets, according to the `OME zarr spec`_.
+
+
+Features
+--------
+
+- Use as a image reader plugin for `napari`_. See `napari-ome-zarr`_.
+- Simple :doc:`cli` to read and download conforming OME-NGFF filesets.
```

Comment:
see comment about the ``Command-line`` [above](https://github.com/ome/ome-zarr-py/pull/121/files#r824742227)